elasticache/redis/redis.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/acorn-io/aws/elasticache"
+	"github.com/acorn-io/aws/elasticache/redis/util"
+	"github.com/acorn-io/services/aws/libs/common"
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsec2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awselasticache"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/constructs-go/constructs/v10"
+	"github.com/aws/jsii-runtime-go"
+	"github.com/sirupsen/logrus"
+)
+
+type redisStackProps struct {
+	awscdk.StackProps
+	ClusterName string
+	UserTags    map[string]string `json:"tags" yaml:"tags"`
+	VpcID       string
+}
+
+func newRedisStack(scope constructs.Construct, props *redisStackProps) (awscdk.Stack, error) {
+	var sprops awscdk.StackProps
+	if props != nil {
+		sprops = props.StackProps
+	}
+
+	// create the stack
+	stack := awscdk.NewStack(scope, jsii.String("redis-stack"), &sprops)
```

Comment:
I think we are titlecasing this id everywhere.

Reply:
Refactored this IDs in the latest commit. However, it's worth noting that these seem to be case insensitive on the AWS side. At least they're displayed as all lower case in the management console.